genrl/deep/common/exploration.py

```diff
@@ -0,0 +1,28 @@
+# create decorator functions for exploration methods
+# decorator takes as param the select_action method and adds noise to it
+# Some exploration methods require networks/access to the model
+import numpy as np
+
+import gym
+
+class EpsilonGreedyDecorator:
```

Comment:
As a user, I would want the decorator name to be short, so can you make this shorter?

Reply:
I think it can just be `epsgreedy`, we dont need `decorator` in the name.